src/main/java/com/ait/lienzo/client/core/shape/wires/SelectionManager.java

```diff
@@ -293,6 +277,36 @@ public boolean onMouseEventBefore(MouseEvent<? extends EventHandler> event)
             return true;
         }
 
+        void drawSelectionShape(final MouseEvent<? extends EventHandler> event)
+        {
+            final double relativeStartX = getSelectionManager().relativeStartX();
+            final double relativeStartY = getSelectionManager().relativeStartY();
+            final double relativeEventX = (event.getX() - getViewportTransform().getTranslateX()) / getViewportTransform().getScaleX();
```

Comment:
mmm quick comment - so it seems fine as we're only using (in Stunner) translation and scaling, but probably it's safer doing this by delegating the transform computation to the `ViewportTransform` instance... because imagine  that we add, for instance, support for canvas/shape rotation, we're missing this parameters here isn't it? 
So what I mean is doing like:
```
final Point2D target = new Point2D(getSelectionManager().relativeStartX(), getSelectionManager().relativeStartY());
getViewportTransform().transform(new Point2D(event.getX(), event.getY()), target);
```
Tbh not sure if that will work hehe, but it should! You should get the same result on the `target` instance... is it true? WDYT? :)

Reply:
Unfortunately, I don't think that the `Transform#transform` method does the same thing that we have here, see [here](https://github.com/kiegroup/lienzo-core/blob/master/src/main/java/com/ait/lienzo/client/core/types/Transform.java#L305-L308) and [here](https://github.com/kiegroup/lienzo-core/blob/master/src/main/java/com/ait/lienzo/client/core/types/Transform.java#L679-L685). In this PR context, I think we need to subtract the `viewport.translateXY` and consider the `viewport.scaleXY`. Am I missing something? :-)